src/Converters.js

```diff
@@ -0,0 +1,124 @@
+export class Converters {
```

Comment:
Is there any attribution needed for where this came from originally?

Reply:
Ah I see the original referenced geotiff - will add this back too